src/Microsoft.MobileBlazorBindings/Extensions/ObjectExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.MobileBlazorBindings.Extensions
+{
+    internal static class ObjectExtensions
+    {
+        /// <summary>
+        /// The only purpose of this method it to use it as delegate parameter.
+        /// </summary>
+        public static object This(this object @this) => @this;
```

Comment:
😢 . I'll merge this for now but I think we'll want to have a different approach, such as a regular static method. I know it's internal, and I know it's in a different namespace. But for those reasons I think this might as well be a regular static method.

Reply:
Filed https://github.com/dotnet/MobileBlazorBindings/issues/448 to think about this more.